src/app/components/dialogs/confirm/confirm.component.scss

```diff
@@ -0,0 +1,16 @@
+@use "../../../themes/colors.scss";
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
```

Comment:
This will probably be used a lot throughout the app.
You should create a class in styles
`.flex-space-between-centered `
to be used by other developers too